api/v1alpha1/managedcluster_types.go

```diff
@@ -91,6 +91,9 @@ type ManagedClusterSpec struct {
 	Priority int32 `json:"priority,omitempty"`
```

Comment:
as mentioned in Slack I expected the Priority per Service and not on the level of the cluster, as it could be that the service has individual priorities. I could also agree to have it in both places, but I feel as the Priority really only is for the Services and the cluster itself does not have a priority I would move it into each of the Service

Reply:
ah, so we put all the defined `services` in a `managedCluster` object into a single sveltos `Profile`? I assumed that we create a separate Profile object per defined  service.

Reply:
Yes each Profile/ClusterProfile can have multiple services. We don't create a separate Profile/ClusterProfile per service.

Reply:
okay, yea I wonder what is better, but lets keep it the way we have it now.
should we then rename `Priority` to `ServicePriority` or something? as right now the word Priority suggest that this is a cluster priority, which it is not, it is only used by Services

Reply:
Changed to `ServicePriority` to better reflect its purpose.

Reply:
ack, change to ServicePriority seems like the right thing.  @Schnitzel all good?  this should solve the last outstanding unresolved item.